docs/mocksupport.rst

```diff
@@ -564,23 +564,37 @@ mode.
 Building the mock repository
 ----------------------------
 
+The mock repository should contain the CIM qualifier declarations, CIM classes,
+CIMInstances, and CIM methods to be used in the mock environment.
+
+:class:`~pywbem_mock.FakedWBEMConnection` and
+:class:`~pywbem_mock.DMTFCIMSchema` provide the tools to build the mock
+repository.
+
+The repository can be build by either defining the mof for the items to be
+inserted into the repository (either by defining the pywbem CIM Objects or by
+defining the MOF and compiling the mof) or by compiling qualifiers and classes
+directly from a DMTF CIM schema. A particular test repository may be a
+combination of classes defined in a DMTF CIM Schema and user classes.
+
+Starting with pywbem 0.13.0, the class :class:`~pywbem_mock.DMTFCIMSchema`
```

Comment:
`FakedWBEMConnection.compile_mof_string` already existed in pywbem 0.12.0. I suggest to omit the mentioning of the version, since correctly attributing the features to pywbem versions would be more complex than helpful (IMO). We have the specific version indicators on each of the features.

Reply:
DONE as part of rewrite of that whole section